src/main/java/seedu/address/logic/parser/FindStarsLessThanCommandParser.java

```diff
@@ -0,0 +1,33 @@
+package seedu.address.logic.parser;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import seedu.address.logic.commands.FindStarsLessThanCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.person.StarsLessThanPredicate;
+
+/**
+ * Parser for the FindStarsLessThanCommand.
+ */
+public class FindStarsLessThanCommandParser implements Parser<FindStarsLessThanCommand> {
+    /**
+     * Parses the given {@code String} of arguments in the context of the FindStarsLessThanCommand
+     * and returns an FindStarsLessThan object for execution.
+     *
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public FindStarsLessThanCommand parse(String args) throws ParseException {
+        requireNonNull(args);
+        try {
+            int intValue = Integer.parseInt(args.replaceAll(" ", ""));
```

Comment:
what would be the expected behaviour if the user inputs `FindStarsLessThan 1 2 3`, would the command throw an error or output all students with less then 123 stars?

Reply:
I was thinking to output all students with less than 123 stars, would it be better to throw an error instead?